content/en/getting_started/integrations/aws.md

```diff
@@ -0,0 +1,300 @@
+
+Depending on which AWS services you're using and your use case for monitoring, there are multiple options within the integration tile to specify the data to be collected. For example, you can limit data collection based on AWS service, namespace, or tags. Additionally, you can choose to mute monitors for instance terminations triggered manually or by autoscaling with [EC2 automuting][12] enabled. If needed, enable [Alarm Collection][13] to send your CloudWatch alarms to the Datadog [Event Stream][14] and choose whether to collect custom metrics.
+
+Datadog uses host statuses reported by the CloudWatch API to distinguish between expected and unexpected instance terminations, preventing unwanted Monitor notifications.
+
+### Validation
+
+View the out-of-the-box [AWS overview dashboard][15] to see metrics sent by your AWS services and infrastructure:
+{{< img src="getting_started/integrations/aws-dashboard.png" alt="An image displaying the AWS overview dashboard in the Datadog account. On the left is the AWS logo and an AWS events graph showing 'No matching entries found'. In the center are graphs related to EBS volumes with numerical data displayed and a heat map showing consistent data. Along the right are graphs related to ELBs showing numerical data as well as a timeseries graph showing spikey data from three sources.">}}
+
+## Exploring other products
+
+### Serverless
+
+You can unify the metrics, traces, and logs from your AWS Lambda functions running serverless applications in Datadog. Check out [Serverless] for instructions on instrumenting your application, installing [Serverless Libraries and Integrations][16], implementing [Distributed Tracing with Serverless Applications][17], or [Serverless Troubleshooting][18].
+
+### Other integrations
+
+<div class="alert alert-warning">
+Datadog's Amazon integration is built to collect <a href="https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/aws-services-cloudwatch-metrics.html">ALL metrics from CloudWatch</a>. Datadog strives to continually update the docs to show every sub-integration, but cloud services rapidly release new metrics and services so the list of integrations are sometimes lagging.
+</div>
+
+| Integration                             | Description                                                                            |
```

Comment:
FYI this list is out of date compared to https://docs.datadoghq.com/integrations/#cat-aws, but likely not worth changing in this iteration.

Reply:
I think that we should link to https://docs.datadoghq.com/integrations/#cat-aws because it is an automatically generated list instead of having the list hard coded here because this is going to be _really hard_ to keep up to date otherwise